corehq/apps/aggregate_ucrs/sql/adapter.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, unicode_literals
+import sqlalchemy
+from memoized import memoized
+
+from corehq.apps.userreports.sql import IndicatorSqlAdapter
+from corehq.apps.userreports.sql.columns import column_to_sql
+from corehq.apps.userreports.util import get_table_name
+
+
+class AggregateIndicatorSqlAdapter(IndicatorSqlAdapter):
```

Comment:
This should hopefully allow us to leverage ~all of UCR framework for table lifecycle management. Though that's currently only a theory. It currently gives us table creation for basically free.